### ISSUE TITLE
Problem running MetaStore standalone in a docker container.

### ISSUE THREAD

**Issue by VolkerHartmann** [kit-data-manager/metastore2#270]
**Describe the bug**
Starting with version 1.2.1, MetaStore throws an error when creating and updating documents, even though everything works fine.

**To reproduce**
1. Start docker container 'docker run -d -p8040:8040 --name standalone  kitdm/metastore2:v1.2.2
2. Register a schema
```
curl -X 'POST' \
  'http://localhost:8040/api/v1/schemas' \
  -H 'accept: application/json' \
  -H 'Content-Type: multipart/form-data' \
  -F 'record=@schema_record.json;type=application/json' \
  -F 'schema=@myschema.json;type=application/json'
could not execute statement          
```

**Expected behavior**
Should return status '201' (CREATED).

**Comment by VolkerHartmann** [comment]
Since version 1.2.1 running MetaStore inside a docker container it expects a RabbitMQ service available.
## Solution 1:
Run the whole framework
```
$ curl -O https://raw.githubusercontent.com/kit-data-manager/metastore2/main/docker-compose.yml
$ docker compose up
```
Metastore is now available at http://metastore.docker:8040
Frontend: http://localhost

## Solution 2:
Adjust the settings:
```
$ mkdir standalone
$ cd standalone
$ cat > application.properties << EOF
###############################################################################
# Messaging - RabbitMQ
###############################################################################
repo.messaging.enabled: false
EOF
$ docker run -d -p8040:8040 --name standalone --mount type=bind,source="$(pwd)",target=/spring/metastore2/config kitdm/metastore2:v1.2.2
```
Metastore is now available at http://localhost:8040

**Comment by VolkerHartmann** [comment]
There is another, simpler solution that is a substitute for solution 2:
## Solution 3:
```
$ docker run -d -p8040:8040 --name standalone -e REPO_MESSAGING_ENABLED=false --name standalone kitdm/metastore2:v1.2.2
```

**Comment by VolkerHartmann** [comment]
Should be fixed in next version.

**Comment by VolkerHartmann** [comment]
# Fixed in version 1.2.3!
